cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -489,6 +488,70 @@ pub(crate) fn low32_will_sign_extend_to_64(x: u64) -> bool {
     xs == ((xs << 32) >> 32)
 }
 
+impl Inst {
+    fn iset_requirement(&self) -> Option<InstructionSet> {
+        match self {
+            // These instructions are part of SSE2, which is a basic requirement in Cranelift, and
+            // don't have to be checked.
```

Comment:
Are all of the below SSE2, e.g. `JmpIf`? (Perhaps they have some options or encodings that are?)

(To be clear, I like this function and the explicitness of this `match`; I'm just curious about the comment is all.)

Reply:
Yep, I just checked the ones for which i had a few doubts:

- the conversions from int to float/doubles (and reciprocally) are SSE (for floats) and SSE2 (for doubles)
- JmpIf is just a plain old conditional jump, (e.g. `jo`), so nothing to fret about.